core/controllers/reader.py

```diff
@@ -194,7 +124,7 @@ def classify_string_classifier_rule(state, normalized_answer):
 # TODO(bhenning): Add more tests for classification, such as testing multiple
```

Comment:
Should this TODO still be here? This PR should introduce frontend tests which satisfy this TODO.